bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -206,14 +205,6 @@ private BigtableOptions(
       RetryOptions retryOptions,
       int timeoutMs,
       int channelCount) {
-    Preconditions.checkArgument(
-        !Strings.isNullOrEmpty(projectId), "ProjectId must not be empty or null.");
```

Comment:
These are all still used below to create the BigtableClusterName. It looks like that constructor also has the same checks in it. In that sense, I'm not sure this really changes much?

Would it make sense to remove project id, zone id, cluster id as direct members of the BigtableOptions class and have all users of those go through BigtableClusterName? (That is probably a bigger change...)